.github/workflows/push-lacchain-environment.yaml

```diff
@@ -116,9 +116,11 @@ jobs:
           HAPI_HASURA_URL: 'http://monitor-hasura:8080/v1/graphql'
```

Comment:
@adriexnet  can we please restore the value for `HAPI_EOS_STATE_HISTORY_PLUGIN_ENDPOINT: ''` in the PR.

Reply:
sure, I will do it

Reply:
done